source/index.html.haml

```diff
@@ -80,4 +80,10 @@
 
     glitchIt();
 
-    var s = skrollr.init();
+    $(function() {
+      var isTouchDevice = "ontouchstart" in document.documentElement;
+
+      if (!isTouchDevice) {
+        var s = skrollr.init();
```

Comment:
I don't know if you need these variables, but your closure is going to isolate them I think.